### PR TITLE
#386 Awards.total() should obey the 90 days rule

### PR DESCRIPTION
**Pull request by fabriciofx**
As per #386 .

### CODE REVIEW

File: src/main/java/com/zerocracy/pmo/Awards.java [hunk 1]
```diff
@@ -113,7 +120,14 @@ public int total() throws IOException {
         try (final Item item = this.item()) {
             return Integer.parseInt(
                 new Xocument(item.path()).xpath(
-                    "sum(/awards/award/points/text())"
+                    String.format(
+                        "sum(/awards/award[added > '%s']/points/text())",
+                        new DateAsText(
+                            // @checkstyle MagicNumberCheck (1 line)
+                            LocalDateTime.now().minusDays(90),
+                            "yyyy-MM-dd"
```
[the remark this answers is not in the document]
Comment (fabriciofx) [reply src/main/java/com/zerocracy/pmo/Awards.java]: @g4s8 You're complete right. I'll change it. Thanks!

File: src/test/java/com/zerocracy/pmo/AwardsTest.java [hunk 2]
```diff
@@ -27,16 +28,38 @@
...
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
 public final class AwardsTest {
 
     @Test
     public void addsAndRemovesPoints() throws Exception {
         final Awards awards = new Awards(new FkProject(), "yegor").bootstrap();
-        awards.add(1, "gh:test/test#1", "just for fun");
-        awards.add(-1, "gh:test/test#2", "just for fun 2");
-        awards.add(1, "gh:test/test#3", "just for fun 3");
+        awards.add(1, "gh:test/test#1", "just for fun 1", LocalDateTime.now());
```
[the remark this answers is not in the document]
Comment (fabriciofx) [reply src/test/java/com/zerocracy/pmo/AwardsTest.java]: @g4s8 I agree. I'll change it now. Thanks!

**Comment by g4s8** [comment]
@fabriciofx please see this comment https://github.com/zerocracy/farm/issues/366#issuecomment-360521152
looks like we need to change implementation logic and remove rewards from `awards.xml` instead of filtering by date in xsl

**Comment by fabriciofx** [comment]
@g4s8 You're right. @yegor256 Should I stop finish this PR? Will you close this issue?

**Comment by yegor256** [comment]
@fabriciofx yes, let's close this PR. We need to remove rewards from the XML, not filter them out during selection

**Comment by fabriciofx** [comment]
@yegor256 So, I'll close this PR. Please, could you close the issue #386 and open another explaining what should be done? @g4s8 Thanks for your support!

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8), total is [+3535](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+8315](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#439](https://github.com/zerocracy/farm/issues/439) is now out of scope

**Pull request closed without merging**